ansible/tasks/postgres-extensions/13-plv8.yml

```diff
@@ -58,4 +64,3 @@
   poll: 0
   register: plv8_build
   when: async_mode
```

Comment:
For async_mode, we need to perform cleanup from [99-finish_async_tasks.yml](https://github.com/supabase/postgres/blob/develop/ansible/tasks/postgres-extensions/99-finish_async_tasks.yml)

Reply:
Good catch - thanks!